build/index.asset.php

```diff
@@ -1 +1 @@
-<?php return array('dependencies' => array('lodash', 'wp-element', 'wp-polyfill'), 'version' => 'b1ed361d4d0e7f58a477b80c455cde99');
\ No newline at end of file
+<?php return array('dependencies' => array('lodash', 'wp-element', 'wp-polyfill'), 'version' => 'a5a3a61e925acbbba98344b12938e3d4');
```

Comment:
No new line here.

Reply:
This is a system generated file. No need for newlines - its best practice but not essential. And in this instance, not possible to force.